index.html

```diff
@@ -105,12 +114,10 @@ <h5 class=" h5 ">Historia del Atletismo</h5>
               
               El seguimiento del atletismo aumentó en Europa y América hasta que en 1896 se iniciaron en Atenas los Juegos Olímpicos de la era moderna, una modificación restaurada de los antiguos juegos que los griegos celebraban en Olímpia. Durante el siglo XX y lo que llevamos del XXI, los juegos se han celebrado en varios países a intervalos de cuatro años, excepto en tiempo de guerra. En 1913 se fundó la Federación Internacional de atletismo Amateur (IAAF), con sede central de Londres, con el fin de ser organismo rector de las competiciones de atletismo a escala internacional, estableciendo las reglas y dando oficialidad a los récords obtenidos por los atletas.
               </p>
-            
-
       </div>
-    </div>
+    
     <!-- Footer -->
-    <footer class="page-footer font-small border">
+    <footer class="page-footer font-small border-top">
```

Comment:
I would change the `page-footer class` to just `footer`(don't forget to update CSS class name if used) since you added the `class` `footer-copyright` right after. It gives class naming consistency.

This is a [must reading](http://getbem.com/naming/) to learn how to name your `classes`.

Reply:
i made some changes but not sure if that what you meant, please can you check again and let me know. 

 <footer class="page-footer font-small border-top">  i need the border-top because i only want one line on top, if that what you point it out that was the only way i know how to do it for now, let me know ok.thanks

Reply:
You should read the article whenever you got time. It's just a better way to declare your CSS classes so your code is more organized, modular & easier to maintain and debug. For now, read the article & when we chat I will go over it with you, just let me know when you got time.